### DISCUSSION TITLE
v-progress-linear does not progress

### DISCUSSION THREAD

**Discussion by john-** [vuetifyjs/vuetify#17164] [Q&A]
I am running vue@3.2.47 and vuetify@3.1.14.  I modified the shell program created by `npm create vue@3` to include a `v-progress-linear` component:
```
[snip]
  <main>
      <div>
        <v-progress-linear
          indeterminate
          color="teal"
        ></v-progress-linear>
      </div>
    <TheWelcome />
  </main>
[snip]
```
However, it does not do the progress animation.  It is only a static light teal bar. with no darker part that moves.

![v-progresss-bar_static](https://user-images.githubusercontent.com/2359125/232342686-1b2f4dfb-5cf9-4f2b-adf7-04a7fe937d09.png)

The determinant version works (controlled via setting value).  Also, the `v-progress-circular` (indeterminate) status works.

Any idea why this is not working for me?

**Comment by yuwu9145** [comment]
Can you provide a full reproduction link/demo?

**Reply by john-** [reply]
My attempts to reproduce it outside of my environment have not been successful.  I will close this ticket until I can provide a demo.
